source/gfoidl.Base64/Internal/Base64Encoder/Base64Encoder.Encode.cs

```diff
@@ -117,7 +117,8 @@ private static void EncodeOneByte<T>(ref byte oneByte, ref T encoded, ref byte e
         }
         //---------------------------------------------------------------------
         // internal because tests use this map too
-        internal static readonly byte[] s_encodingMap = {
+        internal static ReadOnlySpan<byte> EncodingMap => new byte[] {
```

Comment:
This would be the correct name, according to naming rules. Other places have a more "field"-like name, so this should be named `s_encodingMap` (as previous).